core/storage/feedback/gae_models.py

```diff
@@ -88,18 +97,39 @@ def generate_new_thread_id(cls, exploration_id):
 
     @classmethod
     def generate_full_thread_id(cls, exploration_id, thread_id):
+        """Generates a full thread ID which is unique globally
+
+        Args:
+            exploration_id: str,The exploration id the thread belongs to
+            thread_id: str, thread ID of given thread
+
+        Returns: full thread ID by concatenating exploration_id and thread_id
+        """
         return '.'.join([exploration_id, thread_id])
 
     @property
     def thread_id(self):
+        """Splits the full thread id generated in generate_full_thread_id method
```

Comment:
Explain interface, not implementation. "Returns the thread_id (not including the exploration_id) for this model instance."